server/scheduled/send_notifications.php

```diff
@@ -0,0 +1,36 @@
+<?php
+require_once "../functions.php";
+
+
+switch($argv[1]){
+	case "send_booking_number_notifications":
+		$is_web=http_response_code()!==FALSE;
+
+		if(!$is_web) die();
+
+		$start_time = new DateTime('tomorrow', new DateTimeZone($server_default_timezone));
+		$end_time = clone $start_time;
+		$end_time->modify(("+1 day"));
```

Comment:
I know that ther will not be lectures starting at midnight, but technically it is more correct. Check if modify accepts 2 modifiers in the same string.
```suggestion
		$end_time->modify(("+1 day - 1 second"));
```

Reply:
i'm sure it does. For now leave as is it